confgenerator/logging_receivers.go

```diff
@@ -382,10 +382,16 @@ func commonEventLogComponents(useNewerApi bool, channels []string, tag string) [
 	// up blank. The Use_ANSI configuration is provided to work around this; however,
 	// this also strips Unicode characters away, so we only use it on affected
 	// platforms. This only affects the newer API.
-	if useNewerApi && (windows.Is2012() || windows.Is2016()) {
+	if !r.IsDefaultVersion() && (windows.Is2012() || windows.Is2016()) {
 		input[0].Config["Use_ANSI"] = "True"
 	}
 
+	if r.RenderAsXML {
+		input[0].Config["Render_Event_As_XML"] = "True"
+		// Use "raw_xml" because it's a more descriptive name than "System"
```

Comment:
"System" is a constant field name, with no relation to the channel. Will update the comment to clarify where it comes from, since I agree that it can easily be confused with the "System" channel.